### PR TITLE
Meditor 910 history reporting incorrectly

### PR DESCRIPTION
**Pull request by aritrivi**
Fix null modifiedOn date for initial document state. Improve history detail layout in the history panel for readability. History issues are still a potential problem when users "edit"/"save"/"submit for review" from a browser window that is not refreshed and current with any new status changes.

### CODE REVIEW

**Comment by joncarlson** [comment]
Was confused for a second before realizing this is merging from a fork. For anyone else reviewing, here's what I did to use my existing mEditor folder:

git remote add aritrivi https://github.com/aritrivi/gesdisc-meditor.git
git fetch aritrivi
git checkout meditor-910-History-reporting-incorrectly

Just make sure to switch back to your origin branch when you're done

File: packages/app/documents/service.ts [hunk 1]
```diff
@@ -95,11 +95,12 @@ export async function createDocument(
         }
 
         //* This logic (and associated TODO) is ported from Meditor.js, saveDocument. Minimal modifications were made.
-        const rootState = { source: INIT_STATE, target: initialState }
+        const modDate = new Date().toISOString()
```
Comment (joncarlson) [inline packages/app/documents/service.ts:98]: Small nit, took me a bit to figure out what modDate was, may be better spelled out, like `modifiedOnDate`

**Review by joncarlson — APPROVED** [review]
Looks good! Much easier to see the transitions between states. Going ahead and approving, had a small comment

**Review by benforshey — APPROVED** [review]
The history panel looks good, Tony! The `toLocaleString` call is a nice touch. 

It looks like Prettier didn't run on these files, which means that the next time someone makes a change and Prettier does run, it'll add your files to their PR for just formatting changes. You can address that by checking which files aren't formatted correctly:

- From `packages/app` in your terminal, run `npx prettier -l .` This gives you the list of files that don't match our Prettier config.
- For each file, feed those files into Prettier with `npx prettier -w <your-file.extension>` or just YOLO it once with `npx prettier -w .` (this is actually pretty safe since you can just discard changes if any files change that are unrelated to your PR).

**Comment by aritrivi** [comment]
All PR comments addressed.

**Pull request merged**